### ISSUE TITLE
3.1.20: sphinx 4.x warnings

### ISSUE THREAD

**Issue by kloczek** [gitpython-developers/GitPython#1323]
Looks like sphinx 4.x shows some warnings.
```console
+ /usr/bin/python3 setup.py build_sphinx -b man --build-dir build/sphinx
running build_sphinx
Running Sphinx v4.1.2
making output directory... done
building [mo]: targets for 0 po files that are out of date
building [man]: all manpages
updating environment: [new config] 6 added, 0 changed, 0 removed
reading sources... [100%] tutorial
/home/tkloczko/rpmbuild/BUILD/GitPython-3.1.20/doc/source/changes.rst:16: WARNING: Title underline too short.

3.1.19 (YANKED)
======
/home/tkloczko/rpmbuild/BUILD/GitPython-3.1.20/doc/source/changes.rst:16: WARNING: Title underline too short.

3.1.19 (YANKED)
======
looking for now-outdated files... none found
pickling environment... done
checking consistency... done
writing... python-gitpython.3 { intro tutorial reference roadmap changes } done
build succeeded, 2 warnings.
```

**Comment by Byron** [comment]
Thanks for reporting.

It's unclear which version you are using for release, but in `main` [the issue is fixed](https://github.com/gitpython-developers/GitPython/blob/main/doc/source/changes.rst).

Hence I am closing the issue as no action can be performed here.

**Comment by kloczek** [comment]
> It's unclear which version you are using for release, but in `main` [the issue is fixed](https://github.com/gitpython-developers/GitPython/blob/main/doc/source/changes.rst).

`GitPython` version is in the subject (3.1.20). Sphinx version is in the log (4.1.2)

**Comment by kloczek** [comment]
> Hence I am closing the issue as no action can be performed here.

Did you ever try to use command which is in firsr line of the quotesd log?
`/usr/bin/python3 setup.py build_sphinx -b man --build-dir build/sphinx`

**Comment by Byron** [comment]
No.

Are you suggesting there is a fix for this, and if so which one would that be? The headlines shown above don't exist in `main`.

**Comment by kloczek** [comment]
Just FTR .. please I'm not suggestion anything :)
I'm just a messenger which reported warning :P
It may be but in GitPython documentation or sphinx as well (or some sphinx extension).
Sorry to say that but still I have very long list of the python modules to package but on packaging anything all what I'm trying is at least report found issues (please peak on my profile and you will found that it is +90% of my last year github activity).
If I reported that not enough clear that is of course my fault ..

**Comment by Byron** [comment]
Sorry, I didn't mean to come across as condescending or aggressive or be disrespectful towards your work.

Let me express my confusion as to how to fix something that seems already fixed in `main` and that relates to a code snapshot that lies in the past.

Once clarified if this is fixable, the best course of action would be a PR that makes the changes. The cost of this issue seems already much higher than the cost of fixing it if it can be fixed.

**Comment by Yobmod** [comment]
AFAIK I fixed all sphinx warnings (for python <= 3.9), they just haven't been released yet. 
So this should be closable on next release.
(Although sphinx is causing problems with 3.10 still)

**Comment by kloczek** [comment]
OK than feel free to close this toicket and will retest that on next release and if still will be something will tell you :)
